### PR TITLE
Updated GeoWave and JTS versions to the latest release.

### PR DESCRIPTION
**Pull request by jwomeara**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,19 +27,14 @@
         <version.common-utils>1.1</version.common-utils>
         <version.commons-lang3>3.9</version.commons-lang3>
         <version.commons-net>3.6</version.commons-net>
-        <version.geowave>0.9.5</version.geowave>
+        <version.geowave>1.1.0</version.geowave>
         <version.hadoop>3.1.1</version.hadoop>
-        <version.jts>1.13</version.jts>
+        <version.jts>1.16.0</version.jts>
```
Comment (brianloss) [inline pom.xml:32]: Looks like there's a 1.16.1 available. Should we use that instead?

File: pom.xml [hunk 2]
```diff
@@ -207,8 +207,9 @@
             <snapshots>
                 <enabled>false</enabled>
             </snapshots>
-            <id>geowave</id>
-            <url>http://geowave-maven.s3-website-us-east-1.amazonaws.com/release</url>
+            <id>osgeo-release</id>
+            <name>Open Source Geospatial Foundation Repository</name>
+            <url>https://repo.osgeo.org/repository/release/</url>
         </repository>
```
Comment (brianloss) [inline pom.xml:213]: These artifacts are available in maven central, so we should remove this entire repository block and pull from central instead.

File: pom.xml (hunk 1 above)
Comment (jwomeara) [reply pom.xml:32]: I tried to use the one that our version of GeoWave was using to be consistent with them.  I'll update it and if all the tests pass I'll push an update.

File: pom.xml (hunk 2 above)
Comment (jwomeara) [reply pom.xml:213]: I don't think the boundless artifacts are available in maven central.  at least, it wouldn't build for me without this repo.

Comment (brianloss) [reply pom.xml:213]: Wait, so they've deployed artifacts to maven central that have dependencies that aren't in maven central?! That seems like a bug of theirs to me. Sigh.

**Pull request merged**